dotcom-rendering/src/web/lib/decideContainerOverrides.ts

```diff
@@ -325,5 +346,6 @@ export const decideContainerOverrides = (
 		topBar: {
 			card: topBarCard(containerPalette),
 		},
+		divider: divider(containerPalette),
```

Comment:
Would it work if we use `border.container` instead of creating a new property? I presume we'd always want these colours to match?

Reply:
Good question that I'm not sure the answer is to yet!

Reply:
Seems like we should be using border.container!